vault/changelog.early-seed.md

```diff
@@ -10,25 +10,29 @@ nav_exclude: true
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.91.0
+## 0.92.0
 
-Dendron 0.91 has sprouted 🌱
+Dendron 0.92 has sprouted 🌱
```

Comment:
would switch the order:
- graph, tree, self contained vaults
- for graph view, mention the 100x speed increase with 🚀🚀🚀